client/src/views/Dashboard/Dashboard.js

```diff
@@ -78,13 +79,15 @@ export default function Dashboard() {
   const [transactionTotalAmount, transactionTotalAmountLoading] = useGetTransactionTotalAmount()
   const [transactionDailyAverage, transactionDailyAverageLoading] = useGetTransactionDailyAverage()
 
+  const lineChartDataLimiter = isMobileOnly ? 10 : 20
```

Comment:
besides decreasing data pounts count you also could:

- decrease font size
(on line chart)
- rotate X-axis labels by 90º
- decrease line width
- decrease points radius
(on pie chart)
- make labels closer to the outer circle (decreasing diagonal, horizontal length & offset)
- shrink inner circle then shrink whole chart by width

This will allow display a bit more data points on mobile view and diagrams will have to look more proportional

Reply:
Also, lineChartDataLimiter and other mobile-dependend only constants could be moved out of component

Reply:
done